src/admin/importers/BlockImporter.php

```diff
@@ -76,32 +76,81 @@ public function run()
      */
     protected function handleBlockDefinitions(array $definitions)
     {
-        $cwds = [Yii::getAlias('@app'), getcwd()];
+        $directories = array_unique([Yii::getAlias('@app'), getcwd(), null]);
```

Comment:
Why added a null value as last directory item?

Reply:
Yes indeed, my own unit tested brought me to this scenario:

It could be that a block already has the full path, therefore no folder should be prefixed otherwise it won't be found. For example the block `@app/blocks/Xyz.php` would become the @app folder prefixed for 2 times.

Reply:
Ok I see, I already thought something like that. Maybe a comment at that point could be helpfully.

Reply:
I just added an info.